data/MohitBansal.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Mohit Bansal",
+  "type": "personal",
+  "bio": "**Full Stack Web** | **Open Source** enthusiast | Feel Free to reach me out!",
+  "tags": [
+    "Full Stack Web Development",
+    "JavaScript",
+    "Node",
+    "React",
+    "MERN-Stack",
+    "Open Source"
+  ],
+  "socials": [
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/mohitbansalhmh"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/MohitBansal321"
+    },
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/mohit_bansal302"
+    }
+  ],
+  "links": [
+    {
+      "name": "Let's Connect on Linkedin",
+      "url": "https://www.linkedin.com/mohitbansalhmh",
```

Comment:
Linkedin Url is unavailable. You want to correct that.

Reply:
Check it now it is working fine